service/service.go

```diff
@@ -397,8 +397,20 @@ type RunningDriver struct {
 }
 
 func (r *RunningDriver) Start() (string, error) {
-	port := rand.Intn(1000) + 10*1000 // KontainerDriver port range 10,000 - 11,000
-	listenAddress := fmt.Sprintf("%v%v", ListenAddress, port)
+	ephemeralListenAddress := fmt.Sprintf("%s:0", ListenAddress)
```

Comment:
Would be nice to have an inline comment here stating how this works, eg: `Bind to 0 and listen to trigger acquisition of an open port`.